games/d.yaml

```
@@ -1004,6 +1004,30 @@
   updated: 2016-08-25
   url: http://uaf.sourceforge.net/
 
+- name: Dungeon Digger
+  images:
+  - https://a.fsdn.com/con/app/proj/dungeondigger/screenshots/87295.jpg
+  - https://a.fsdn.com/con/app/proj/dungeondigger/screenshots/85468.jpg
+  - https://a.fsdn.com/con/app/proj/dungeondigger/screenshots/96980.jpg
+  - https://a.fsdn.com/con/app/proj/dungeondigger/screenshots/108567.jpg
+  - https://a.fsdn.com/con/app/proj/dungeondigger/screenshots/84862.jpg
+  lang:
+  - C++
+  - Lua
+  framework:
+  - SDL
+  license:
+  - GPL3
```

```suggestion
  - GPL2
```
example: https://sourceforge.net/p/dungeondigger/code/HEAD/tree/digger/trunk/src/GameInterface.cpp

---

Fixed.